public/app/features/provisioning/Wizard/BootstrapStep.tsx

```diff
@@ -94,12 +81,27 @@ export function BootstrapStep({ settingsData, repoName }: Props) {
                   noMargin
                   {...field}
                 >
-                  <Card.Heading>{action.label}</Card.Heading>
+                  <Card.Heading>
+                    <Text variant="h5">{action.label}</Text>
+                  </Card.Heading>
                   <Card.Description>
+                    <div className={styles.divider} />
```

Comment:
We have a [component](https://developers.grafana.com/ui/latest/index.html?path=/story/layout-divider--basic) for this.